### DISCUSSION TITLE
How can I update this module when a mouse action is performed?

### DISCUSSION THREAD

**Discussion by Samueru-sama** [polybar/polybar#2978] [Q&A]
I'm using this module to display the volume in decibels, I know that Polybar already has a default pulseaudio module that does the same, however due to the lack of customization, specially not being able to round the results from for example 5.23 dB to 5 dB I ended up making this instead: 

```
[module/VOLUME]
type = custom/script
label = %output%
exec = pactl list sinks | awk '/State: /{flag=1} /dB/ && flag {printf("%.0f dB", $7); exit} {if ($1 == "Mute:" && $2 == "yes") printf(" Muted")}'
interval = 0.5
format-padding = 1
scroll-up = pulsemixer --change-volume +5 --max-volume 100 
scroll-down = pulsemixer --change-volume -5 --max-volume 100 
click-left = pgrep pavucontrol && killall pavucontrol || pavucontrol
click-right = pactl set-sink-mute @DEFAULT_SINK@ toggle
```

The only issue I have is the interval time, even at 0.5 it is not very responsive when I use the scroll wheel to change the volume, is there a way that I could make this module update every time a mouse action is performed?

**Comment by patrick96** [comment]
This is not possible right now with interval-based scripts, this feature is tracked in #786

There is an example of update-on-click for a custom date module on the [wiki](https://github.com/polybar/polybar/wiki/Module:-script#examples) for a script with `tail = true`.
